src/pages/library/DataBrowserPreview.js

```diff
@@ -1,7 +1,7 @@
+import ReactJson from '@microlink/react-json-view';
 import _ from 'lodash/fp';
```

Comment:
Before and after, tested that copy worked.

![image](https://github.com/DataBiosphere/terra-ui/assets/484484/7e6ae49c-bcfd-4c0f-ab62-090db5fe4405)
![image](https://github.com/DataBiosphere/terra-ui/assets/484484/44d2c45c-61ec-426b-9dc7-0e4a36713fd9)